course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -352,12 +440,14 @@
             };
             $http.delete(courseMemberURL, config)
                 .success(function(data, status, headers, config, statusText) {
-                    var success = membership; // TODO - copy to avoid stomping the original?
+                    // make profile available to template for success message
+                    var success = angular.copy(membership);
                     success.searchTerm = $scope.getProfileFullName(membership.profile)
                         || membership.user_id;
-                    success.action = 'removed from';
+                    success.type = 'remove';
+                    success.alertType = 'success';
                     $scope.clearMessages();
-                    $scope.success = success;
+                    $scope.messages.success = success;
```

Comment:
Not worth redoing, but would having a `$scope.setSuccessMessage(success)` method be a cleaner approach here?  So it's not possible to forget to do `clearMessages()` first?